scheduler.py

```diff
@@ -14,28 +14,33 @@ def __init__(self):
         self.completion_status = {}
         self.workerConnectInfoLock = threading.Lock()
         self.workerIdLock = threading.Lock()
-        self.workers = {} # Map that maintains the worker id to hostName,portNum 
+        self.workers = {} # Map that maintains the worker id to hostName,portNum
         self.globalIncrementalWorkerId = 0; # A globally incrementing worker id maintained by the scheduler
         signal.signal(signal.SIGINT, self.sigterm_handler)
 
     def submit_task(self, task : Task) -> Future:
-        random_worker = random.choice(self.workers)
+        if self.globalIncrementalWorkerId > 1:
+            random_worker_id = random.randint(0, self.globalIncrementalWorkerId - 1)
+        else:
+            random_worker_id = 0
+        print(random_worker_id)
```

Comment:
Can please you convert this to a logging stmt or remove if not needed?

Reply:
done